Mathlib/Topology/Defs/Filter.lean

```diff
@@ -211,7 +211,6 @@ def MapClusterPt {ι : Type*} (x : X) (F : Filter ι) (u : ι → X) : Prop :=
 See also `ClusterPt`. -/
 def AccPt (x : X) (F : Filter X) : Prop :=
   NeBot (𝓝[≠] x ⊓ F)
-#align acc_pt AccPt
```

Comment:
Why do you drop an `#align`?

Reply:
I have no clue, I must've accidentally done this and missed that this change was made, I'll revert that!